abs/replica_client.go

```diff
@@ -53,41 +61,100 @@ func (c *ReplicaClient) Init(ctx context.Context) (err error) {
 	c.mu.Lock()
 	defer c.mu.Unlock()
 
-	if c.containerURL != nil {
+	if c.client != nil {
 		return nil
 	}
 
-	// Read account key from environment, if available.
-	accountKey := c.AccountKey
-	if accountKey == "" {
-		accountKey = os.Getenv("LITESTREAM_AZURE_ACCOUNT_KEY")
-	}
-
-	// Authenticate to ACS.
-	credential, err := azblob.NewSharedKeyCredential(c.AccountName, accountKey)
-	if err != nil {
-		return err
+	// Validate required configuration
+	if c.Bucket == "" {
+		return fmt.Errorf("abs: container name is required")
 	}
 
 	// Construct & parse endpoint unless already set.
 	endpoint := c.Endpoint
 	if endpoint == "" {
+		if c.AccountName == "" {
+			return fmt.Errorf("abs: account name is required when endpoint is not specified")
+		}
 		endpoint = fmt.Sprintf("https://%s.blob.core.windows.net", c.AccountName)
 	}
-	endpointURL, err := url.Parse(endpoint)
-	if err != nil {
-		return fmt.Errorf("cannot parse azure endpoint: %w", err)
+
+	// Check if we have explicit credentials or should use default credential chain
+	accountKey := c.AccountKey
+	if accountKey == "" {
+		accountKey = os.Getenv("LITESTREAM_AZURE_ACCOUNT_KEY")
 	}
 
-	// Build pipeline and reference to container.
-	pipeline := azblob.NewPipeline(credential, azblob.PipelineOptions{
-		Retry: azblob.RetryOptions{
-			TryTimeout: 24 * time.Hour,
-		},
-	})
-	containerURL := azblob.NewServiceURL(*endpointURL, pipeline).NewContainerURL(c.Bucket)
-	c.containerURL = &containerURL
+	// Create Azure Blob Storage client with appropriate authentication
+	var client *azblob.Client
+	if accountKey != "" && c.AccountName != "" {
+		// Use shared key authentication (existing behavior)
+		credential, err := azblob.NewSharedKeyCredential(c.AccountName, accountKey)
+		if err != nil {
+			return fmt.Errorf("cannot create shared key credential: %w", err)
```

Comment:
nit, for consistency:
```suggestion
			return fmt.Errorf("abs: cannot create shared key credential: %w", err)
```